static/templates/search_operators.hbs

```diff
@@ -173,7 +173,7 @@
                 {{#*inline "z-operator"}}<p><span class="operator">{{> @partial-block}}</span></p>{{/inline}}
                 {{#*inline "z-stream-prefix"}}stream:{{/inline}}
                 {{#*inline "z-stream"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
-                {{#*inline "z-sender-prefix"}}email:{{/inline}}
+                {{#*inline "z-sender-prefix"}}sender:{{/inline}}
```

Comment:
I think there's also a connected bug in the paragraph before this; `email` inside the `z-email` block should be the sample email address, `user@example.com`.